string action_id
rosmc_interface_msgs/ActionStatus updated_status
---
string error_msg